packages/manager/modules/vps/src/routing.js

```diff
@@ -55,6 +70,12 @@ export default /* @ngInject */ ($stateProvider) => {
               }
               return true;
             }),
+        tabSummary: /* @ngInject */ (serviceName, VpsService) => {
+          const forceRefresh = true;
+          return VpsService.getTabSummary(serviceName, forceRefresh);
+        },
+        vps: /* @ngInject */ (serviceName, VpsService) =>
```

Comment:
Doesn't dashboard routing inherit from this ? In that case duplication isn't needed https://github.com/ovh/manager/pull/2344/files#diff-bda40ae7c79147d9e97de02cea2bad2eR59

Reply:
Nice catch, I forgot to delete it in vps routing